### ISSUE TITLE
Page titles should be unique

### ISSUE THREAD

**Issue by dsamojlenko** [cds-snc/c19-benefits-node#208]
Pages all have the same `<title>`.  Each page should have its own unique and descriptive `<title>`; perhaps the `<h1>` text on each page could be pre-pended to the current `<title>`.

**Comment by CalvinRodo** [comment]
Page title's should also be localized right now they display only in English.
